src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -304,6 +304,7 @@ class BeamSim @Inject()(
 
     logger.info("Ending Iteration")
     delayMetricAnalysis.generateDelayAnalysis(event)
+    beam.analysis.AnalysisProcessor.iterationEndBulkAnalysisOutput_Async(event.getIteration)
```

Comment:
probably better to pass the iteration output path than just the iteration

Reply:
Done